.eslintrc.cjs

```diff
@@ -10,9 +10,7 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   plugins: ['react-refresh'],
   rules: {
-    'react-refresh/only-export-components': [
-      'warn',
-      { allowConstantExport: true },
-    ],
+    'react-refresh/only-export-components': ['warn', { allowConstantExport: true },],
+    '@typescript-eslint/no-explicit-any': 'warn',
```

Comment:
Doesn't signal the use of any as error

Reply:
@afonsobspinto can you explain why you changed this.

Reply:
The linting command was erroring because we are defining some types as any.
To be fair, the above change doesn't fully fix the problem. On top of that I had to run:
`eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 40` (be aware of the 40 instead of 0).
The any were already in the codebase.

I won't mind rolling back, but we (the team) need to be aware of it and code accordingly

Reply:
Sounds good to me. Maybe it would be nice to have a github action that runs the linting command and if it fails we don't approve the PR. Otherwise we end up in a situation like the current one where we have 28 `any` warnings.

Reply:
yes we should do that for all projects :-)

if you feel confortable doing it, please go ahead @afonsobspinto 
if not then no problem I can also do it

Reply:
As hopefully a one of, I replaced all the `any` with real types. 
I also played a little bit with https://github.com/super-linter/super-linter but I couldn't make it work with our own .eslintrc.cjs. I think we can merge this PR like this and address the github action in a new card. Do you agree?
I can be assigned to the new task no problem. I like to automate stuff 👍